### DISCUSSION TITLE
Tube vs Discharge Hematocrit

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#38] [General discussion]
Hello All!

Again me! OK here is the big issue I am having regarding the hematocrit value. I mentioned that @gzavo, you have used discharge hematocrit to compare your data with Pries exp. data in your 2017 paper. I am talking about Fig. 7. So isn't computing the volume fraction of the RBCs inside the tube being considered tube hematocrit? Why and how have you computed discharge hematocrit in there?

Can you please explain that a little? That would be a Greeaaat help! I have been stuck in there. Most appreciated.

Best,
Mojtaba

**Comment by mojtabaAmiraslanpour** [comment]
@gzavo Do you have any comments about that? I can't actually figure it out. My simulation data fits well with the experiment when I am using Hd. Why is that?

**Comment by gzavo** [comment]
Dear @mojtabaAmiraslanpour ,

This is not really a HemoCell related question. Your best source of information is the already mentioned Pries '92 paper, which explains it well. Briefly: in a tube you have higher velocity and higher RBC density in the middle, hence the effective amount of RBCs passing through in a given time is higher than what you would get based on the local hematocrit and flow rate.

**Reply by mojtabaAmiraslanpour** [reply]
@gzavo Well, I am calculating hematocrit by computing the volume fraction of RBCs in the pipe. That way I get what my Ht (Tube Hematocrit) is. So then I compare the resulting rel. app. viscosity and the corresponding Ht value with experimental data (which is provided by Pries 1992). But that does not give me the correct results.

But When I interpret the computed H as Hd (Discharge Hematocrit) everything is good. I cannot understand why I should consider the volume fraction of RBCs as Hd.

**Comment by mojtabaAmiraslanpour** [comment]
Any comments?

**Comment by gzavo** [comment]
You don't really need further comments, as you mostly answered your own question. The difference between tube and tank hematocrit is not that one is in a tube and the other is in a tank, but one relates to _moving_ blood with non-constant flow profile, the other to _standing still_.

**Comment by mojtabaAmiraslanpour** [comment]
I am afraid that I am not able to convey what I am asking for. I already know what you are saying. Let me ask my question in a step-by-step manner. It would make more sense this way. This formula:

![image](https://user-images.githubusercontent.com/15649053/156428971-aa08da29-bbf9-48ec-b886-f4e89bae4356.png)

As far as I know, this would give us the tube hematocrit. Then we would easily compute the discharge hematocrit by using this one:

![image](https://user-images.githubusercontent.com/15649053/156429549-279d40ed-41d2-4b75-bbdc-709f023ba76e.png)

Are we on the same page up to here?

2nd Q: Can you also please tell me what do you consider for the volume of a single RBC when computing the Hematocrit value?

**Reply by gzavo** [reply]
Your second eq. is a very generic definition of discharge hematocrit, and is only true under the assumption of homogeneous cell distributions (i.e., not in a vessel). There were several mentions of the Pries '92 paper already, so please read it for details on how to calculate discharge hematocrit in straight pipes (eq. 8 and eq. 10).
As already discussed in a previous question you asked, RBC volume is 81.116 um3, but when you aim for viscosity arising from collisions you need to include the IBM kernel size, and compute with an effective RBC volume of 97.687 um3.
So for Q2: 81.116 um3

**Reply by mojtabaAmiraslanpour** [reply]
Thank you @gzavo. 
Pardon my repetitive questions. I am trying to gain a good understanding of the simulation parameters so I can validate my results. I have read the Pries paper and will do it again to see if I have missed the concept.

**Reply by gzavo** [reply]
Hi @mojtabaAmiraslanpour,
Did you manage to solve this eventually?

**Reply by mojtabaAmiraslanpour** [reply]
To be honest, not yet!

My final goal is to be able to validate the pipeflow case with Pries Rel. App. Vis. data. Ht definition (The volume fraction of cells) and Re number would significantly change the results in my cases and yet I have not been able to find a final combination of parameters that would work for a range of pipe diameters.

The latest problem that I have faced is the volume conservation issue. I get more than a ~10%-20% change in the average volume of RBCs after about 0.5s of simulation time. I expected it to be less than 1%. I mentioned tau_v = 0.01 which is large enough to cause that. I used tau_v=0.0001 (corresponding to ~1% change in volume) and successfully crashed my case!

**Comment by mojtabaAmiraslanpour** [comment]
So I am back again since I have not been able to find the answer!
The thing is that we need to validate the Hd obtained by the simulation with the Pries '92 eqns 8-10. I see in many related literature Hd from the simulation is obtained by the same equation that I just mentioned above. So I cannot understand why it is wrong to use that and why you mention that we need to calculate it from the same experimental equations.
